config/config_test.go

```diff
@@ -0,0 +1,235 @@
+package config
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/BurntSushi/toml"
+	"github.com/stretchr/testify/assert"
+)
+
+func testLookupEnv(key string) (string, bool) {
+	mockEnv := map[string]string{
+		"GORAY":         "some_value",
+		"THREADS_COUNT": "5",
+	}
```

Comment:
better to keep it as a global var, later we can probably use it in another functions where we will ned the same mock data